pkg/cmd/server/admin/create_mastercerts.go

```diff
@@ -136,6 +139,9 @@ func (o CreateMasterCertsOptions) Validate(args []string) error {
 	if len(o.SignerName) == 0 {
 		return errors.New("signer-name must be provided")
 	}
+	if o.ExpireDays < 0 {
```

Comment:
If 0 is not a valid value you should error it, setting 0 and then finding out it was defaulted is not a good UX.